### ISSUE TITLE
Missing SA1212 on init accessor

### ISSUE THREAD

**Issue by bjornhellander** [DotNetAnalyzers/StyleCopAnalyzers#3652]
```
public class TestClass
{
    public int X { set; get; } // Triggers SA1212
    public int Y { init; get; } // Does NOT trigger SA1212!
}
```

I expect SA1212 ("A get accessor appears after a set accessor within a property or indexer") to trigger on both properties above, but it does not. That is, I expect `init` and `set` to be treated the same way in this rule.
Tested with 1.2.0-beta.435.
